### DISCUSSION TITLE
Change the color of the STEREO icon to yellow in the base interface theme

### DISCUSSION THREAD

**Discussion by Yurij20** [PE5PVB/TEF6686_ESP32#401] [Ideas]
There is a proposal to change the color of the STEREO icon to yellow in the basic interface theme.  Red doesn't look very good, especially at minimum display brightness

**Comment by PE5PVB** [comment]
Well, if someone wants it, just add an extra theme with that color set. I don't like it either.

**Comment by PE5PVB** [comment]
Won't be changed in the default theme, I don't like it. If someone wants it, they are free to add their own theme to the repository.
